src/main/kotlin/frc/robot/subsystems/ReefCommands.kt

```diff
@@ -89,7 +89,7 @@ fun outtakeCoral(): Command =
                 ))
             .withTimeout(0.5),
         gripper.slowOuttake(true).withTimeout(0.1),
-        moveDefaultPosition().onlyIf(gripper.hasCoral.negate())
+        moveDefaultPosition().onlyIf(gripper.hasCoral.negate().and{!DriverStation.isAutonomous()})
```

Comment:
Use [`RobotModeTriggers.teleop()`](https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj2/command/button/RobotModeTriggers.html#teleop()) instead:
```suggestion
            .and(RobotModeTriggers.teleop())
```
Why is that change needed?